torch/fx/experimental/proxy_tensor.py

```diff
@@ -331,15 +331,16 @@ def can_handle_tensor(x):
         return r
 
     # For pre-autograd tracing, we do not want to run CompositeImplicit decomps.
-    if not pre_dispatch and func not in [
```

Comment:
Why are we removing this check? (if pre_dispatch is False and `export_inference` is False, we will now decompose?